workers/unity/Assets/Gdk/Core/GameObjectRepresentation/GameObjectTranslationBase.cs

```diff
@@ -0,0 +1,28 @@
+using Unity.Entities;
+
+namespace Improbable.Gdk.Core
+{
+    public abstract class GameObjectTranslationBase
```

Comment:
Best to call this something other than translation. It doesn't look like this is translating anything, just defining groups and doing some piping.

Reply:
Renamed to `GameObjectComponentDispatcher`.